src/lib/rest/ConnectionInfo.cpp

```diff
@@ -211,7 +210,28 @@ ConnectionInfo::~ConnectionInfo()
   servicePathV.clear();
 }
 
-void ConnectionInfo::modify(const std::string &_url, const std::string &_verb, const std::string &_payload, const HttpHeaders &head)
+void ConnectionInfo::reset()
```

Comment:
I understand that both modify() and reset() methods has been included in the ConnectionInfo for the feature/1181_websockets and are used from the ws library. It would be great if you could provide a function preamble for each of them, in order to provide some documentation about the purpose of the method, how it is used, etc. I doesn't need to be too much detailed, but explain at least a little about the method.

E.g (in this same file):

```
`/* ****************************************************************************
*
* uriParamTypesParse - parse the URI param 'type' into uriParamTypes vector
*/
```